src/style.css

```diff
@@ -1 +1,86 @@
-/* styles go here */
+.fieldset_info {
```

Comment:
Why you need all of .fieldset_... classes, please provide only one and use it in each fieldset. Try do not repeat yourselt!